src/current/v23.2/monitoring-and-alerting.md

```diff
@@ -288,63 +298,62 @@ curl -X POST http://localhost:8080/_status/critical_nodes
+To introduce a violation that you can then query for, you'll [set the table locality to `REGIONAL BY TABLE`]({% link {{ page.version.version }}/alter-table.md %}#set-the-table-locality-to-regional-by-table) for the [`movr.users`]({% link {{ page.version.version }}/movr.md %}#the-movr-database) table.
+
+You can use [`SHOW CREATE TABLE`]({% link {{ page.version.version }}/show-create.md %}) to see what existing [table locality]({% link {{ page.version.version }}/table-localities.md %}) is attached to the `users` table, so you know what to modify.
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
 SHOW CREATE TABLE users;
 ~~~
 
 ~~~
-  table_name |                                    create_statement
--------------+-----------------------------------------------------------------------------------------
+  table_name |                     create_statement
+-------------+-----------------------------------------------------------
   users      | CREATE TABLE public.users (
              |     id UUID NOT NULL,
              |     city VARCHAR NOT NULL,
              |     name VARCHAR NULL,
              |     address VARCHAR NULL,
              |     credit_card VARCHAR NULL,
              |     CONSTRAINT users_pkey PRIMARY KEY (city ASC, id ASC)
-             | ) PARTITION BY LIST (city) (
-             |     PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-             |     PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-             |     PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-             | );
-             | ALTER PARTITION europe_west OF INDEX movr.public.users@users_pkey CONFIGURE ZONE USING
-             |     constraints = '[+region=europe-west1]';
-             | ALTER PARTITION us_east OF INDEX movr.public.users@users_pkey CONFIGURE ZONE USING
-             |     constraints = '[+region=us-east1]';
-             | ALTER PARTITION us_west OF INDEX movr.public.users@users_pkey CONFIGURE ZONE USING
-             |     constraints = '[+region=us-west1]'
+             | ) LOCALITY REGIONAL BY TABLE IN PRIMARY REGION
 (1 row)
 ~~~
 
-To create a constraint violation, use the [`ALTER PARTITION`]({% link {{ page.version.version }}/alter-partition.md %}) statement to tell the [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) in the `europe_west` partition that they are explicitly supposed to *not* be in the `region=europe-west1` locality:
+To create a constraint violation, use the [`ALTER TABLE ... SET LOCALITY`]({% link {{ page.version.version }}/alter-table.md %}#set-the-table-locality-to-regional-by-table) statement to move the [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) in the `PRIMARY REGION` (`us-east1`) that they are explicitly supposed to move to the `europe-west1` locality:
```

Comment:
This sentence needs rewriting, it's confusing b/c it says "to move the ranges ... that they are explicitly supposed to move" which doesn't make sense:

> To create a constraint violation, use the ALTER TABLE ... SET LOCALITY statement to move the ranges in the PRIMARY REGION (us-east1) that they are explicitly supposed to move to the europe-west1 locality

Suggest something like:

> To create a constraint violation, use the ALTER TABLE ... SET LOCALITY statement to tell the ranges in the PRIMARY REGION (us-east1) that they are supposed to be in the europe-west1 locality

or if you don't like the "tell the ranges" language maybe something like

> To create a constraint violation, use the ALTER TABLE ... SET LOCALITY statement to configure the ranges in the PRIMARY REGION (us-east1) to be in the europe-west1 locality